source/compliance.txt

```diff
@@ -6,48 +6,189 @@ Compliance
+its security, privacy, and organizational controls. These audits help ensure
+that the platform  meets compliance and regulatory requirements, including
+those specific to highly-regulated industries.
 
-      .. include:: /includes/shared-settings-clusters-devtest.rst
+{+service+} has ISO/IEC 27001 and 9001, PCI DSS, HIPAA, GDPR, FedRAMP, and
+several other certifications. The full list of certifications is included
+in this section.
 
-   .. tab:: Staging and Prod Environments
-      :tabid: stagingprod
+These certifications serve as a foundation for meeting
+industry-specific regulatory requirements. Use these certifications
+to maintain security controls and ensure that your data practices align
+with necessary compliance demands.
 
-      .. include:: /includes/shared-settings-clusters-stagingprod.rst
+{+service+} maintains the following certifications:
 
-.. tabs::
+- ISO/IEC 27001:2022: Global standard for information security management systems.
+- ISO/IEC 27017:2015: Global standard for cloud-specific security controls.
+- ISO/IEC 27018:2019: Global standard for protecting sensitive data (PII) on the cloud.
+- ISO 9001:2015: Globally recognized standard for Quality Management.
+- PCI DSS: Requirements for processing and accessing credit card data.
+- HIPAA: U.S. privacy regulation safeguarding health information.
+- GDPR: General Data Protection Regulation in the EU and EEA.
+- IRAP: Australian government cybersecurity assessment framework.
+- TX-RAMP: Texas Risk and Authorization Management Program.
+- TISAX: Trusted Information Security Assessment Exchange for automotive industries.
+- HDS: French standard for hosting health data (Hébergeur de Données de Santé).
 
-   .. tab:: CLI
-      :tabid: cli
+To learn more, visit the |atlas-trust|, a comprehensive compliance and
+trust program for MongoDB {+service+}.
+
+.. _arch-center-compliance-encryption:
+
+Encryption
+~~~~~~~~~~
 
-      Content here
+Implement encryption to ensure data security and compliance across all
+stages of data handling.
 
-   .. tab:: Terraform
-      :tabid: Terraform
+- By default, |service| encrypts data in transit. To ensure data privacy and
+  regulatory compliance, |service| requires |tls-ssl| to encrypt the connections
+  to your databases. In addition, you can configure SSL or TLS OCSP certificate
+  revocation checking. To learn more,
+  see :atlas:`OCSP Certificate Revocation Check <oscp-cert-check>`.
 
-      Content here
+- By default, |service| encrypts all data at rest. |service| uses |AES-256|
+  encryption to encrypt all data stored in |s3| buckets in your |service|
+  {+database-deployments+}.
+  In addition, |service| supports using |aws| |kms|, Azure Key Vault, and |gcp| to
+  encrypt storage engines and cloud provider backups. To learn more, see
+  :atlas:`Encryption at Rest using your Key Management 
+  <security-kms-encryption>`. 
+
+- Use :manual:`queryable encryption </core/queryable-encryption/>` to secure
```

Comment:
Reframe this as "you can" because this page isn't making recommendations (compliance recs will be in a future version)
```suggestion
- You can use :manual:`queryable encryption </core/queryable-encryption/>` to secure
```